protocol/src/protocol.semanticHighlighting.proposed.md

```diff
@@ -0,0 +1,89 @@
+#### Semantic Highlighting
+
+While the syntax highlighting are done on the client-side and can handle keywords, strings, and other low-level tokens from the grammar, it cannot adequately support complex coloring. Semantic highlighting information is calculated on the language server and pushed to the client as a notification. This notification carries information about the ranges that have to be colored. The desired coloring details are given as [TextMate scopes](https://manual.macromates.com/en/language_grammars) for each affected range. For the semantic highlighting information the following additions are proposed:
+
+_Client Capabilities_:
+
+Capability that has to be set by the language client if that can accept and process the semantic highlighting information received from the server.
+
+```ts
+/**
+ * The text document client capabilities.
+ */
+workspace: {
+
+	/**
+	 * `true` if the client has the semantic highlighting support for the text document. Otherwise, `false`. It is `false` by default.
```

Comment:
`true` if the client has the semantic highlighting support for the text document.
...to...
`true` if the client supports semantic highlighting support text documents.